packages/web-react/config/webpack.js

```diff
@@ -19,7 +19,7 @@ module.exports = (env, argv) =>
     plugins: [
       new BundleAnalyzerPlugin({
         statsFilename: 'browser-stats.json',
-        generateStatsFile: true,
+        generateStatsFile: false,
```

Comment:
Do we need this plugin at all? Right now it won't do anything, right? The generating is disabled and the analyzer too. Maybe we don't need it?